_posts/2016-09-30-rimdev-io-going-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "RIMdev.io Going Open"
+date: 2016-09-30 12:04:59
+tags: 
+- RIMdev
+- Team
+- Jekyll
+categories: Blog
+twitter_text: "RIMdev.io going open"
+authors: Khalid Abuhakmeh
+image: https://farm1.staticflickr.com/3/4014689_a1bbcaf037_b_d.jpg
+image_url: https://www.flickr.com/photos/velkr0/
+image_credit: velkr0
+---
+
+Our team believes in being open and transparent, and while we write consistently with some amazing content, we are equally as proud of what we've done with the Jekyll platform. Based on our philosophy of a team being open, transparent, and known as contributors we have decided to make our site a public repository on [GitHub](https://github.com/ritterim/ritterim.github.io). 
```

Comment:
I suggest beginning the post as `Our team believes in being open and transparent. And, while` or just `While`.